core/src/main/java/org/axonframework/commandhandling/disruptor/CommandHandlerInvoker.java

```diff
@@ -295,5 +313,26 @@ private void removeFromCache(String aggregateIdentifier) {
                 }
             }
         }
+
+        @SuppressWarnings("Duplicates")
+        @Override
+        public void send(Message<?> message, ScopeDescriptor scopeDescription) throws Exception {
+            if (canResolve(scopeDescription)) {
+                String aggregateIdentifier = ((AggregateScopeDescriptor) scopeDescription).getIdentifier().toString();
+                Aggregate<T> aggregate = load(aggregateIdentifier);
+                if (aggregate != null) {
```

Comment:
Note that the load method throws `AggregateNotFoundException` instead of returning `null` when an aggregate isn't found.
Please create a test case to verify this behavior.